packages/commercetools/theme/tests/e2e/integration/e2e-breadcrumbs.spec.ts

```diff
@@ -0,0 +1,40 @@
+import page from '../pages/factory';
+
+const testsCases = [
+  'category',
+  'subcategory',
+  'nested subcategory'
+];
+
+const breadcrumbsNames = [
+  'Home',
+  'Women',
+  'Clothing',
+  'Jackets'
+];
+
+context('Breadcrumbs', () => {
```

Comment:
```suggestion
context(['regression], 'Breadcrumbs', () => {
```
Please add 'regression' tag to context :)